src/main/webapp/app/shared/components/card/partials/card-thumbnail.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren, useEffect, useRef, useState } from 'react';
+import { View, Image } from '@adobe/react-spectrum'
+
+interface ICardThumbnailProps {
+  thumbnail: React.ReactNode;
+}
+const CardThumbnail: React.FC<PropsWithChildren<ICardThumbnailProps>> = props => {
+
+  return (
+    <>
+      <View>
```

Comment:
Not everything has to be in view component. Use only when necessary like when you want to set some configuration